Bigtable/src/DataClient.php

```diff
@@ -480,12 +441,12 @@ public function sampleRowKeys(array $options = [])
      *           either `trueMutations` or `falseMutations` must be provided.
      * }
      *
-     * @return bool Returns true if predicate filter yielded any output, false otherwise.
-     * @throws ApiException if the remote call fails or operation fails
-     * @throws \InvalidArgumentException if neither of $trueMutations or $falseMutations is set.
-     *         if $predicateFilter is not instance of {@see Google\Cloud\Bigtable\Filter\FilterInterface}.
-     *         if $trueMutations is set and is not instance of {@see Google\Cloud\Bigtable\Mutations}.
-     *         if $falseMutations is set and is not instance of {@see Google\Cloud\Bigtable\Mutations}.
+     * @return bool Returns true If predicate filter yielded any output, false otherwise.
```

Comment:
Thanks for hitting all of these, but this looks like the victim of a rogue ctrl+f :)

Reply:
done, that was victim of distraction :)